app/services/rate_limit_checker.rb

```diff
@@ -15,7 +16,8 @@ class RateLimitChecker
     rate_limit_published_article_creation: { min: 0, placeholder: 9, description: "The number of articles a user can create within 30 seconds" },
     rate_limit_image_upload: { min: 0, placeholder: 9, description: "The number of images a user can upload within 30 seconds" },
     rate_limit_email_recipient: { min: 0, placeholder: 5, description: "The number of emails we send to a user within 2 minutes" },
-    rate_limit_organization_creation: { min: 1, placeholder: 1, description: "The number of organizations a user can create within a 5 minute period" }
+    rate_limit_organization_creation: { min: 1, placeholder: 1, description: "The number of organizations a user can create within a 5 minute period" },
```

Comment:
This constant only seems to be used in `app/views/internal/configs/show.html.erb`, should we maybe move it to a view helper?

Reply:
That would be a good idea going forward